pkg/leaderelection/lease_pool_test.go

```diff
@@ -176,10 +176,10 @@ func TestSecondWatcherAcquiresReleasedLease(t *testing.T) {
 			Name: "test",
 		},
 		Spec: coordinationv1.LeaseSpec{
-			HolderIdentity:       pointer.String("pool1"),
+			HolderIdentity:       ptr.To("pool1"),
 			AcquireTime:          &now,
 			RenewTime:            &now,
-			LeaseDurationSeconds: pointer.Int32(60 * 60), // block lease for a very long time
+			LeaseDurationSeconds: ptr.To(int32((1 * time.Hour).Seconds())), // block lease for a very long time
```

Comment:
I think just doing `ptr.To(int32(time.Hour.Seconds()))` would be a bit cleaner. But it's certainly not worth a discussion.